lib/jsxgettext.js

```diff
@@ -116,7 +119,12 @@ function parse(sources, options) {
       throw err;
   }
 
-  options.keyword = options.keyword || ['gettext'];
+  if( options.keyword ) {
```

Comment:
Looks like a combined if block is the better way to go here instead of an if block + `options.keyword || ` approach